pkg/util/sqlkiller/sqlkiller_test.go

```
@@ -0,0 +1,55 @@
+
+import (
+	"testing"
+
+	"github.com/pingcap/tidb/pkg/testkit/testfailpoint"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSQLKillerConcurrentReset(t *testing.T) {
+	t.Run("reset after successful kill signal CAS", func(t *testing.T) {
+		killer := &SQLKiller{}
+		testfailpoint.EnableCall(t, "github.com/pingcap/tidb/pkg/util/sqlkiller/afterSendKillSignalCAS", func() {
```

golangci-lint (2.12.2)

[warning] line-length-limit: line is 113 characters, out of limit 80

(revive)